src/components/Team/MemberCard.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { BrowserRouter as Router, Route } from "react-router-dom";
+import "../../utility/fonts.css";
+import styled from 'styled-components';
+import SubpageOuter from "../SubpageOuter/SubpageOuter.jsx";
+import devices from '../../utility/MediaQueries.js';
+import {StaticP, StaticH1} from "../../utility/ContentStyles.js";
+import './MemberCard.css';
+
+
+
+var cardStyle = {
```

Comment:
Best practice is to use `let` rather than `var` since `let`'s behavior is almost always the one we want